integrated_channels/integrated_channel/admin/__init__.py

```diff
@@ -16,7 +16,7 @@ class ContentMetadataItemTransmissionAdmin(admin.ModelAdmin):
         'enterprise_customer',
         'integrated_channel_code',
         'content_id',
-        'channel_metadata'
```

Comment:
modified is useful, the giant metadata object is not useful on the list view, slows the performance of this page down